### DISCUSSION TITLE
Add ticks in the past

### DISCUSSION THREAD

**Discussion by frrk** [iSoron/uhabits#2039] [Feature Requests] [answered]
I have a habit that I have to complete once every four weeks. I forgot to tick it two weeks ago, but my screen does not let me go back 14 days to add the completed habit.

Would be nice if I could add a status for a custom date. Or, allow horizontal scrolling.

**Comment by amrsamii** [comment, marked as answer]
You can. Click on the habit and in the calendar section, press edit, then you can check any past day!

**Comment by ezio-melotti** [comment]
> Or, allow horizontal scrolling.

Note that horizontal scrolling is already available (albeit it's not too obvious): try dragging the top bar (with the day/date) left and right.
